name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/settings/AttributeFieldType.java

```diff
@@ -25,7 +27,8 @@
     QUOTE(Long.class, QuoteConverter.class), //
     SHARE(Long.class, ShareConverter.class), //
     DATE(LocalDate.class, DateConverter.class), //
-    BOOLEAN(Boolean.class, BooleanConverter.class);
+    BOOLEAN(Boolean.class, BooleanConverter.class), //
+    LIMIT_PRICE(LimitPrice.class, LimitPriceConverter.class);
```

Comment:
Mit dieser Änderung kann ein ```LimitPrice``` Attribut an **alle** Typen angehängt werden - nicht nur an ein Wertpapier sondern auch an ein Konto, Depot oder einen Sparplan. Es macht aber nur Sinn an einem Konto. Um das auseinander zu steuern gibt es im AttributeType ein ```target```.

Sowas brauchen wir jetzt auch im ```AttributeFieldType``` damit das ```AttributeListTab``` nur die passenden Attribute vorschlägt - also ein Filter in [AttributeListTab.java#L103](https://github.com/buchen/portfolio/blob/ba8b86647f858ff16216cc504ef48dd0b6990c6e/name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/settings/AttributeListTab.java#L103).

Wie könnte man das machen? Vielleicht hier bei der enum noch ein dritter (optionaler) Parameter der, wenn gesetzt, bedeutet dass das Attribut nur für diesen target type angelegt werden kann? ```LIMIT_PRICE(LimitPrice.class, LimitPriceConverter.class, Security.class)```

Reply:
Dieser dritte Parameter ist aber vielleicht auch wieder etwas unschön, wenn man ein AttributeFieldType an nur zwei Typen angehängt werden kann. Also diese Beschränkung auf nur einen Typen würde mich etwas stören.

Reply:
Fehlt eigentlich nur noch eine gute Lösung für dieses Problem...